packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -39,6 +39,9 @@ export const styles = {
       pointerEvents: 'none', // Disable link interactions
       cursor: 'default',
     },
+    '@media print': {
+      WebkitPrintColorAdjust: 'exact',
```

Comment:
Should we rather switch to [`color-adjust`](https://www.w3.org/TR/css-color-adjust-1/#perf) which is on standard-track? The result should be the same since `jss` should convert this to `--webkit-print-color-adjust`.

Reply:
JSS supports the auto-prefixing but styled-components and emotion don't. Should we stick to the webkit approach for broader support with v5?

Looking at where the prefixer of sc and emotion are implemented, I could find https://github.com/thysultan/stylis.js/blob/master/src/Prefixer.js. For JSS it's in https://github.com/cssinjs/css-vendor/blob/71dc0fd1bb73d9b1b1da836b3a73bfcc67b3f268/src/plugins/color-adjust.js.

I have opened this issue https://github.com/thysultan/stylis.js/issues/210.

Reply:
Why does it matter what other libraries do? They don't consume these styles, no?

Reply:
Works with [autoprefixer](https://autoprefixer.github.io/) and fixes this behavior in firefox as well.

Reply:
@eps1lon I think until a property is standardized it should not be implemented. This can cause certain issues across certain browsers.

Reply:
What type of issues? I though CSS was designed not to break when facing unknown rules?

Reply:
@oliviertassinari it depends. if you use https://github.com/NanoHttpd/nanohttpd in a webview invalid css throws an error and crashes it. ( example for non standard web usages on the css engine )

Reply:
I confused this with  invalid css stylesheets. For rules it should be fine.

Reply:
> I think until a property is standardized it should not be implemented.

We use a bunch of css properties that aren't standardized. This doesn't even apply here because `color-adjust` is standardized while `-webkit-print-color-adjust` isn't. `color-adjust` just hasn't made it beyond the "editors draft" stage. 

> if you use NanoHttpd/nanohttpd in a webview invalid css throws an error and crashes it

That is pretty bold of them. They have to stay on top of any property that is being standardized or works in any browser.

Reply:
> Unknown properties. User agents must ignore a declaration with an unknown property

https://www.w3.org/TR/CSS21/syndata.html

Reply:
@j-mendez As a workaround, you can use our theme.overrides API to remove this CSS prop.